packages/ember-core/src/components/scaffold.gts

```diff
@@ -154,19 +155,26 @@ export default class Scaffold extends Component<ScaffoldSignature> {
               </ContextMenuItem>
             {{/if}}
             <Modal
+              class="subtle"
               @isOpen={{this.showAboutModel}}
               @onDismiss={{fn this.toggleAboutModal false}}
             >
-              <:header>
-                {{t "nrg.app-bar.about.title"}}
-              </:header>
               <:default>
-                <div class="p-3">
-                  <span>
-                    {{this.environmentConfig.modulePrefix}}
-                    ({{version}})
-                  </span>
-                </div>
+                {{#if (has-block "about")}}
+                  {{yield to="about"}}
+                {{else}}
+                  <div class="p-3">
+                    <h4>
+                      {{this.environmentConfig.modulePrefix}}
+                    </h4>
+                    <span>
+                      {{t "nrg.app-bar.about.versionPrefix"}}
+                    </span>
+                    <span>
+                      {{version}}
+                    </span>
```

Comment:
Do these need to be in different spans? If not, we can move the version into the translation:
```suggestion
                    <span>
                      {{t "nrg.app-bar.about.version" version=(version)}}
                    </span>
```
```json
"version": "Version {version}"
```